core/server/helpers/ghost_head.js

```diff
@@ -279,6 +281,23 @@ function finaliseSchema(schema, head) {
     return head;
 }
 
+function getAjaxHelper() {
+    var ghostUrlScript = fs.readFileSync(path.join(config.paths.corePath, 'shared', 'ghost-url.js'), 'utf8'),
+        template = hbs.compile(ghostUrlScript, path.join(config.paths.subdir || '/', 'shared', 'ghost-url.js')),
+        apiPath = '/ghost/api/v0.1/',
+        url, useOrigin;
+
+    if (config.forceAdminSSL) {
+        url = 'https://' + (config.urlSSL || config.url).replace(/.*?:\/\//g, '').replace(/\/$/, '') + apiPath;
+        useOrigin = false;
+    } else {
+        url = config.paths.subdir + apiPath;
+        useOrigin = true;
+    }
+
+    return '<script type="text/javascript">' + template({url: url, useOrigin: useOrigin ? 'true' : 'false'}) + '</script>';
```

Comment:
@ErisDS this `template()` call doesn't seem to want to render correctly on my local.

`console.log(url)` the line before prints out the url and then the api string, so `/blog/ghost/api/v0.1/` (my subdir is set to blog).

However, all that I get in the actual rendered template is `/blog/`.